status/worksets.py

```diff
@@ -109,6 +123,13 @@ def post(self, workset):
         note = self.get_argument('note', '')
         user = self.get_secure_cookie('user')
         email = self.get_secure_cookie('email')
+        category = self.get_argument('category', '')
```

Comment:
Should we set categories to 'Workset' (or 'Flowcell') when it syncs to the Project Running notes? We could use the value set by the user if the user sets a value and default to 'Workset' (or 'Flowcell') otherwise.

Reply:
Fixed now!